frontend/src/views/CanteensPage/ContactForm.vue

```diff
@@ -1,55 +1,49 @@
 <template>
   <div>
     <h2 class="font-weight-black text-h6 grey--text text--darken-4 mb-6">Contactez « {{ canteen.name }} »</h2>
-    <div class="text-left grey--text text--darken-2 mt-n1 mb-6">
-      <p>Vous pouvez par exemple :</p>
-      <div class="ml-2">
-        <p class="mb-1">
-          <v-icon small color="green">$check-line</v-icon>
-          demander des informations supplémentaires sur la cantine que vous fréquentez;
+    <div class="text-left grey--text text--darken-4 mt-n1 mb-6 text-body-2">
+      <div>
+        <p class="mb-1 d-flex align-center">
+          <v-icon small color="green" class="mr-1">$check-line</v-icon>
+          Demandez des informations supplémentaires sur la cantine « {{ canteen.name }} »
         </p>
-        <p class="mb-1">
-          <v-icon small color="green">$check-line</v-icon>
-          contacter un collègue gestionnaire afin de lui demander un retour d'expérience sur une action mise en place...
+        <p class="mb-1 d-flex align-center">
+          <v-icon small color="green" class="mr-1">$check-line</v-icon>
+          Demandez aux gestionnaires leur retour d'expérience sur une action mise en place
         </p>
       </div>
-      <p class="mt-4">Il ne s'agit cependant pas de :</p>
-      <div class="ml-2">
-        <p class="mb-1">
-          <v-icon small color="red">$close-line</v-icon>
-          postuler à une offre d'emploi pour une cantine => privilégier des contacts de l'établissements hors de ma
-          cantine
+      <div class="mt-4">
+        <p class="mb-1 d-flex align-center">
+          <v-icon small color="amber darken-3" class="mr-1">$error-warning-line</v-icon>
+          Privilégiez les contacts directs de l'établissement pour postuler à une offre d'emploi
```

Comment:
I liked the original structure which put the user need up front, making it a bit easier to scan the list to identify where your need is.
Maybe these sentences can be rewritten like:
pour X, faire Y
e.g.
"pour postuler à une offre d'emploi, privilégiez..."
"pour informer de l'absence ou présence de soi ou d'un tiers, utilisez la vie scolaire"

Reply:
Reformulation ici : https://github.com/betagouv/ma-cantine/pull/2257/commits/594c8a3cdf567acfc1244b5497ea89d9d923ee76